pyartcd/pyartcd/pipelines/verify_release.py

```
@@ -0,0 +1,247 @@
+
+            # Process results (individual steps already logged their status)
+            for step_result in step_results:
+                result.steps.append(step_result)
+
+        return result
+
+    async def _safe_run_step(self, step_name: str, step_fn):
+        """Wrap step execution to catch all exceptions and return FAIL instead."""
+        try:
+            return await step_fn()
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)